### DISCUSSION TITLE
Picture of my suggestion

### DISCUSSION THREAD

**Discussion by jere357** [community/community#50978] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

![github2](https://user-images.githubusercontent.com/22777913/227678429-aabe81a3-a8d5-46f3-8203-f9316094f84f.png)
i haven't been able to figure out how to do this, if it' even possible. I haven't seen a single profile without it

**Comment by radar-blip** [comment, marked as answer]
Lots of people want to remove that. It's not possible at the moment. 😕

**Reply by jere357** [reply]
okay, hopefully it will be implemented in the future, ty
